harness/determined/core/_train.py

```diff
@@ -67,11 +68,12 @@ def _get_last_validation(self) -> Optional[int]:
         logger.debug(f"_get_last_validation() -> {steps_completed}")
         return steps_completed
 
-    def report_training_metrics(
+    def report_trial_metrics(
         self,
-        steps_completed: int,
-        metrics: Dict[str, Any],
-        batch_metrics: Optional[List[Dict[str, Any]]] = None,
+        metric_type: str,
+        total_batches: int,
```

Comment:
why was `steps_completed` renamed to `total_batches`?

Reply:
we have removed traces of `step`s in our system and there is still open work to remove the last remaining mentions. this is a move toward that.